src/utils/DecoyDatabase.cpp

```diff
@@ -59,6 +60,11 @@ using namespace std;
   This allows you to specify your target database plus a contaminant file and obtain a concatenated
   target-decoy database using a single call, e.g., DecoyDatabase -in human.fasta crap.fasta -out human_TD.fasta
 
+  If a combined database is requested (default), i.e. @em only_decoy is not used), then Target and Decoy sequences
+  are written interleaved. You need all Targets before the Decoy for some reason, use @em only_decoy and concatenate the files
```

Comment:
Again, unusual capitalization of "Targets" and "Decoy" (should be "decoys", plural).